tests/drivers/i2s/i2s_speed/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,61 @@
+/* i2s-node0 is the transmitter/receiver */
+
+#include <zephyr/dt-bindings/memory-attr/memory-attr-arm.h>
+/ {
+	aliases {
+		i2s-node0 = &i2s1;
+		i2s-node1 = &i2s2;
+	};
+};
+
+&i2s1 {
+	pinctrl-0 = <&i2s1_ws_pa4 &i2s1_ck_pa5 &spi1_miso_pa6 &spi1_mosi_pb5>;
+	pinctrl-names = "default";
+	interrupts = <55 3>;
+	dmas = <&gpdma1 0 7 STM32_DMA_PERIPH_TX
+		&gpdma1 1 6 (STM32_DMA_PERIPH_TO_MEMORY | STM32_DMA_PERIPH_NO_INC |STM32_DMA_MEM_INC | STM32_DMA_PERIPH_16BITS |STM32_DMA_MEM_16BITS | STM32_DMA_PRIORITY_HIGH)>;
+	dma-names = "tx", "rx";
+	status = "okay";
+};
+
+&i2s2 {
+	pinctrl-0 = <&i2s2_ws_pa3 &i2s2_ck_pi1 &spi2_miso_pi2 &spi2_mosi_pb15>;
+	pinctrl-names = "default";
+	interrupts = <56 2>;
+	dmas = <&gpdma1 0 9 (STM32_DMA_MEMORY_TO_PERIPH | STM32_DMA_PERIPH_NO_INC |STM32_DMA_MEM_INC | STM32_DMA_PERIPH_16BITS |STM32_DMA_MEM_16BITS | STM32_DMA_PRIORITY_HIGH)
+		&gpdma1 1 8 STM32_DMA_PERIPH_RX>;
+	dma-names = "tx", "rx";
+	status = "okay";>>
+};
+
+&i2s3 {
+	pinctrl-0 = <&i2s3_ws_pa15 &i2s3_ck_pb3 &spi3_miso_pb4 &spi3_mosi_pc12>;
+	pinctrl-names = "default";
+	interrupts = <57 2>;
+	dmas = <&gpdma1 0 11 (STM32_DMA_MEMORY_TO_PERIPH | STM32_DMA_PERIPH_NO_INC |STM32_DMA_MEM_INC | STM32_DMA_PERIPH_16BITS |STM32_DMA_MEM_16BITS | STM32_DMA_PRIORITY_HIGH)
+		&gpdma1 1 10 STM32_DMA_PERIPH_RX>;
+	dma-names = "tx", "rx";
+	status = "disabled";
+};
+
+&spi1 {
+	status = "disabled";
+};
+
+&spi2 {
+	status = "disabled";
+};
+
+&spi3 {
+	status = "disabled";
+};
+
+&gpdma1 {
+	interrupts = <27 1 28 2 29 1 30 1 31 1 32 1 33 1 34 1>;
```

Comment:
A comment would be nice to highlight that you're basically modifying prio here.
This being said, is that strictly required as most other priorities are 0 (which maybe we can discuss and review globally in another PR) ?